deployer-test.sh

```diff
@@ -16,6 +16,7 @@ LDAP_PROVIDERS="[{'name': 'test', 'challenge': 'true', 'login': 'true', 'kind':
 MASTER_IP=10.35.48.200
 OPENSHIFT_ANSIBLE_REF=openshift-ansible-3.6.140-1
 OPENSHIFT_ANSIBLE_REPO_URL=https://github.com/openshift/openshift-ansible
+PLUGIN_URLS="https://github.com/moolitayer/ocp-ansible-jenkins/blob/master/plugins/osh_get_secret.sh https://github.com/moolitayer/ocp-ansible-jenkins/blob/master/plugins/osh_get_secret.sh"
```

Comment:
It should support more then one URL. Here I arbitrarily tested with the same URL twice. Does not make sense in real life but ok for testing